src/QVClient.ts

```diff
@@ -197,4 +222,28 @@ export class QVClient {
     const uri = new XC.UrlBuilder(plainUri);
     return this._http.postAsync(uri, {});
   }
+
+  public PostStartQuickPoll(message: string): Promise<void> {
+    var shortId = this._sheetId.substr(3);
+    let plainUri = `/election/${shortId}/ajaxquickpoll?msg=${message}`;
+    const uri = new XC.UrlBuilder(plainUri);
+    return this._http.postAsync(uri, {});
+  }
+
+  public PostCloseQuickPoll(): Promise<void> {
+    var shortId = this._sheetId.substr(3);
+    let plainUri = `/election/${shortId}/ajaxquickpoll/done`;
+    const uri = new XC.UrlBuilder(plainUri);
+    return this._http.postAsync(uri, {});
+  }
+
+  public PostSubmitResults(body: {
+    round: number;
+    results: ITallyResultsEntry[];
+  }): Promise<void> {
+    var shortId = this._sheetId.substr(3);
```

Comment:
nit: can we factor this expression to its own function, like: 
 this.GetShortId();